packages/library-base/src/commonMain/kotlin/io/realm/kotlin/types/RealmSet.kt

```diff
@@ -0,0 +1,5 @@
+package io.realm.kotlin.types
+
+public interface RealmSet<E>
```

Comment:
™️ sentence incoming: it is done in the next PR 😅

I won't change it just yet since I'm working against a feature branch so it will be reviewed before it goes into master.